bukkit/src/main/java/me/william278/husksync/bukkit/util/PlayerSetter.java

```diff
@@ -160,7 +160,25 @@ public static void setPlayerFrom(Player player, PlayerData dataToSet) {
             // Set the player's data from the PlayerData
             try {
                 if (Settings.syncAdvancements) {
-                    setPlayerAdvancements(player, DataSerializer.deserializeAdvancementData(data.getSerializedAdvancements()), data);
+                    ArrayList<DataSerializer.AdvancementRecord> advancementRecords
+                            = DataSerializer.deserializeAdvancementData(data.getSerializedAdvancements());
+
+                    if (Settings.useNativeImplementation)
```

Comment:
Change the formatting on this a bit too, try to stick to [google's code style](https://google.github.io/styleguide/javaguide.html).

```java
if (Settings.useNativeImplementation) {
    // ...
} else {
    // ...
}
```

Reply:
Please export the .editorConfig file, for code style

Reply:
![image](https://user-images.githubusercontent.com/90945793/147280666-ce00b3d6-3f9c-4938-abb5-6b171eceb9f0.png)

Reply:
Export and publish to your repository for other developers

Reply:
Sure, can be done.

Reply:
https://drive.google.com/file/d/1_694N7UUw4tf9ixDSiKmnpp4tjXI4EPt/view?usp=sharing